### PR TITLE
FRDQ-10 - Create HomePage

### PR DESCRIPTION
**Pull request by PiotrGrobelak**
I leave some comments in code for the check.  
Persons from UI can check results ;) 
Core questions:  I can't overwrite or add styles for core components with a styled(component) solution.

### CODE REVIEW

**Comment by hayuna** [comment]
<p>Total Coverage: <code>57.58</code></p>
<details><summary>Coverage report</summary>
<p>
<pre>
> frontend@0.1.0 test:coverage /home/runner/work/frontend/frontend
> npm test -- --coverage


> frontend@0.1.0 test /home/runner/work/frontend/frontend
> CI=true react-scripts test --env=jsdom "--coverage"

------------------------------------------|----------|----------|----------|----------|-------------------|
File                                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
------------------------------------------|----------|----------|----------|----------|-------------------|
All files                                 |    57.58 |    85.71 |    45.45 |    57.58 |                   |
 src                                      |        0 |      100 |        0 |        0 |                   |
  App.tsx                                 |        0 |      100 |        0 |        0 |                 6 |
  index.tsx                               |        0 |      100 |      100 |        0 |                 7 |
  react-app-env.d.ts                      |        0 |        0 |        0 |        0 |                   |
 src/components/ExampleStorybookComponent |        0 |      100 |        0 |        0 |                   |
  ExampleStorybookComponent.model.ts      |        0 |        0 |        0 |        0 |                   |
  ExampleStorybookComponent.tsx           |        0 |      100 |        0 |        0 |              9,10 |
 src/components/atoms/Button              |    90.91 |    90.91 |      100 |    90.91 |                   |
  Button.model.ts                         |        0 |        0 |        0 |        0 |                   |
  Button.tsx                              |    90.91 |    90.91 |      100 |    90.91 |                51 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/Header              |      100 |      100 |      100 |      100 |                   |
  Header.tsx                              |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/List                |        0 |      100 |        0 |        0 |                   |
  List.model.ts                           |        0 |        0 |        0 |        0 |                   |
  List.tsx                                |        0 |      100 |        0 |        0 |          12,13,16 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/ListItem            |        0 |        0 |        0 |        0 |                   |
  ListItem.model.ts                       |        0 |        0 |        0 |        0 |                   |
  ListItem.tsx                            |        0 |        0 |        0 |        0 |             11,12 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SnackBar            |      100 |      100 |      100 |      100 |                   |
  SnackBar.tsx                            |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SocialButton        |      100 |      100 |      100 |      100 |                   |
  SocialButton.model.ts                   |        0 |        0 |        0 |        0 |                   |
  SocialButton.tsx                        |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/pages/Home                           |        0 |      100 |        0 |        0 |                   |
  Home.tsx                                |        0 |      100 |        0 |        0 |             22,23 |
  data.ts                                 |        0 |      100 |      100 |        0 |                 1 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/styles                               |       75 |      100 |      100 |       75 |                   |
  GlobalStyles.tsx                        |        0 |      100 |      100 |        0 |                 4 |
  colors.ts                               |      100 |      100 |      100 |      100 |                   |
  devices.ts                              |      100 |      100 |      100 |      100 |                   |
  typography.ts                           |      100 |      100 |      100 |      100 |                   |
------------------------------------------|----------|----------|----------|----------|-------------------|
</pre>
</p>
</details>

**Comment by hayuna** [comment]
<p>Total Coverage: <code>57.58</code></p>
<details><summary>Coverage report</summary>
<p>
<pre>
> frontend@0.1.0 test:coverage /home/runner/work/frontend/frontend
> npm test -- --coverage


> frontend@0.1.0 test /home/runner/work/frontend/frontend
> CI=true react-scripts test --env=jsdom "--coverage"

------------------------------------------|----------|----------|----------|----------|-------------------|
File                                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
------------------------------------------|----------|----------|----------|----------|-------------------|
All files                                 |    57.58 |    85.71 |    45.45 |    57.58 |                   |
 src                                      |        0 |      100 |        0 |        0 |                   |
  App.tsx                                 |        0 |      100 |        0 |        0 |                 6 |
  index.tsx                               |        0 |      100 |      100 |        0 |                 7 |
  react-app-env.d.ts                      |        0 |        0 |        0 |        0 |                   |
 src/components/ExampleStorybookComponent |        0 |      100 |        0 |        0 |                   |
  ExampleStorybookComponent.model.ts      |        0 |        0 |        0 |        0 |                   |
  ExampleStorybookComponent.tsx           |        0 |      100 |        0 |        0 |              9,10 |
 src/components/atoms/Button              |    90.91 |    90.91 |      100 |    90.91 |                   |
  Button.model.ts                         |        0 |        0 |        0 |        0 |                   |
  Button.tsx                              |    90.91 |    90.91 |      100 |    90.91 |                51 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/Header              |      100 |      100 |      100 |      100 |                   |
  Header.tsx                              |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/List                |        0 |      100 |        0 |        0 |                   |
  List.model.ts                           |        0 |        0 |        0 |        0 |                   |
  List.tsx                                |        0 |      100 |        0 |        0 |          12,13,16 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/ListItem            |        0 |        0 |        0 |        0 |                   |
  ListItem.model.ts                       |        0 |        0 |        0 |        0 |                   |
  ListItem.tsx                            |        0 |        0 |        0 |        0 |             11,12 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SnackBar            |      100 |      100 |      100 |      100 |                   |
  SnackBar.tsx                            |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SocialButton        |      100 |      100 |      100 |      100 |                   |
  SocialButton.model.ts                   |        0 |        0 |        0 |        0 |                   |
  SocialButton.tsx                        |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/pages/Home                           |        0 |      100 |        0 |        0 |                   |
  Home.tsx                                |        0 |      100 |        0 |        0 |             22,23 |
  data.ts                                 |        0 |      100 |      100 |        0 |                 1 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/styles                               |       75 |      100 |      100 |       75 |                   |
  GlobalStyles.tsx                        |        0 |      100 |      100 |        0 |                 4 |
  colors.ts                               |      100 |      100 |      100 |      100 |                   |
  devices.ts                              |      100 |      100 |      100 |      100 |                   |
  typography.ts                           |      100 |      100 |      100 |      100 |                   |
------------------------------------------|----------|----------|----------|----------|-------------------|
</pre>
</p>
</details>

File: src/pages/Home/Home.styled.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+const buttonsStyle = css`
+  flex-direction: row;
+  height: auto;
+  div:first-child {
+    margin-right: 16px;
+  }
+`;
+
+// Is the possibility another way to set same styles for few views? Something like device.tablet + device.desktop {styles}?
```
Comment (hayuna) [inline src/pages/Home/Home.styled.ts:68]: You could add new rule to styles/devices with your resolutions

File: src/pages/Home/Home.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import HeroImage from 'assets/hero.svg';
+import SocialButton from 'components/atoms/SocialButton';
+import listItemIcon from 'assets/listItemIcon.svg';
```
Comment (hayuna) [inline src/pages/Home/Home.tsx:4]: Hmm Maybe we can create index.tsx for all assets and export them by name?

Comment (PiotrGrobelak) [reply src/pages/Home/Home.tsx:4]: index.ts

```
import HomePageImg_1 from './homepage-image-1.svg';
import listItemIcon from './listItemIcon.svg';

export { HomePageImg_1, listItemIcon };
```

File: src/pages/Home/Home.styled.ts (hunk 1 above)
Comment (PiotrGrobelak) [reply src/pages/Home/Home.styled.ts:68]: Hmm, I have a good question. What pattern do we want to use? Mobile-first? 


Ok, I will be thinking about how to solve my issue.

**Comment by hayuna** [comment]
<p>Total Coverage: <code>57.58</code></p>
<details><summary>Coverage report</summary>
<p>
<pre>
> frontend@0.1.0 test:coverage /home/runner/work/frontend/frontend
> npm test -- --coverage


> frontend@0.1.0 test /home/runner/work/frontend/frontend
> CI=true react-scripts test --env=jsdom "--coverage"

------------------------------------------|----------|----------|----------|----------|-------------------|
File                                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
------------------------------------------|----------|----------|----------|----------|-------------------|
All files                                 |    57.58 |    85.71 |    45.45 |    57.58 |                   |
 src                                      |        0 |      100 |        0 |        0 |                   |
  App.tsx                                 |        0 |      100 |        0 |        0 |                 6 |
  index.tsx                               |        0 |      100 |      100 |        0 |                 7 |
  react-app-env.d.ts                      |        0 |        0 |        0 |        0 |                   |
 src/assets                               |        0 |        0 |        0 |        0 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/ExampleStorybookComponent |        0 |      100 |        0 |        0 |                   |
  ExampleStorybookComponent.model.ts      |        0 |        0 |        0 |        0 |                   |
  ExampleStorybookComponent.tsx           |        0 |      100 |        0 |        0 |              9,10 |
 src/components/atoms/Button              |    90.91 |    90.91 |      100 |    90.91 |                   |
  Button.model.ts                         |        0 |        0 |        0 |        0 |                   |
  Button.tsx                              |    90.91 |    90.91 |      100 |    90.91 |                51 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/Header              |      100 |      100 |      100 |      100 |                   |
  Header.tsx                              |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/List                |        0 |      100 |        0 |        0 |                   |
  List.model.ts                           |        0 |        0 |        0 |        0 |                   |
  List.tsx                                |        0 |      100 |        0 |        0 |          13,14,17 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/ListItem            |        0 |        0 |        0 |        0 |                   |
  ListItem.model.ts                       |        0 |        0 |        0 |        0 |                   |
  ListItem.tsx                            |        0 |        0 |        0 |        0 |             11,12 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SnackBar            |      100 |      100 |      100 |      100 |                   |
  SnackBar.tsx                            |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SocialButton        |      100 |      100 |      100 |      100 |                   |
  SocialButton.model.ts                   |        0 |        0 |        0 |        0 |                   |
  SocialButton.tsx                        |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/pages/Home                           |        0 |      100 |        0 |        0 |                   |
  Home.tsx                                |        0 |      100 |        0 |        0 |             19,20 |
  data.ts                                 |        0 |      100 |      100 |        0 |                 1 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/styles                               |       75 |      100 |      100 |       75 |                   |
  GlobalStyles.tsx                        |        0 |      100 |      100 |        0 |                 4 |
  colors.ts                               |      100 |      100 |      100 |      100 |                   |
  devices.ts                              |      100 |      100 |      100 |      100 |                   |
  typography.ts                           |      100 |      100 |      100 |      100 |                   |
------------------------------------------|----------|----------|----------|----------|-------------------|
</pre>
</p>
</details>

File: src/pages/Home/Home.tsx (hunk 2 above)
Comment (hayuna) [reply src/pages/Home/Home.tsx]: assets/images/index.ts
```
export { default as HomePageImg_1 }  from './homepage-image-1.svg';
export { default as ListItemIcon } from './listItemIcon.svg';
```

Could you check it?

Comment (hayuna) [reply src/pages/Home/Home.tsx]: [https://github.com/mui-org/material-ui/blob/master/packages/material-ui-icons/src/index.js](https://github.com/mui-org/material-ui/blob/master/packages/material-ui-icons/src/index.js)

Comment (PiotrGrobelak) [reply src/pages/Home/Home.tsx]: ok

File: src/components/atoms/ListItem/ListItem.styled.ts [hunk 3]
```diff
@@ -13,6 +13,9 @@ export const Item = styled.li`
   p {
     display: inline;
   }
+  p {
```
Comment (hayuna) [inline src/components/atoms/ListItem/ListItem.styled.ts:16]: Could we merge these paragraphs?

File: src/components/atoms/Header/Header.styled.ts [hunk 4]
```diff
@@ -1,14 +1,19 @@
-import styled from 'styled-components'
+import styled from 'styled-components';
 
 import device from 'styles/devices';
 import typography from 'styles/typography';
 import colors from 'styles/colors';
 
 export const Title = styled.h1`
+  /* margin-bottom: 32px; */
```
Comment (hayuna) [inline src/components/atoms/Header/Header.styled.ts:8]: Please remove these comments before review

**Comment by hayuna** [comment]
<p>Total Coverage: <code>48.48</code></p>
<details><summary>Coverage report</summary>
<p>
<pre>
> frontend@0.1.0 test:coverage /home/runner/work/frontend/frontend
> npm test -- --coverage


> frontend@0.1.0 test /home/runner/work/frontend/frontend
> CI=true react-scripts test --env=jsdom "--coverage"

------------------------------------------|----------|----------|----------|----------|-------------------|
File                                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
------------------------------------------|----------|----------|----------|----------|-------------------|
All files                                 |    48.48 |    85.71 |    36.36 |    48.48 |                   |
 src                                      |        0 |      100 |        0 |        0 |                   |
  App.tsx                                 |        0 |      100 |        0 |        0 |                 6 |
  index.tsx                               |        0 |      100 |      100 |        0 |                 7 |
  react-app-env.d.ts                      |        0 |        0 |        0 |        0 |                   |
 src/assets                               |        0 |        0 |        0 |        0 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/ExampleStorybookComponent |        0 |      100 |        0 |        0 |                   |
  ExampleStorybookComponent.model.ts      |        0 |        0 |        0 |        0 |                   |
  ExampleStorybookComponent.tsx           |        0 |      100 |        0 |        0 |              9,10 |
 src/components/atoms/Button              |    90.91 |    90.91 |      100 |    90.91 |                   |
  Button.model.ts                         |        0 |        0 |        0 |        0 |                   |
  Button.tsx                              |    90.91 |    90.91 |      100 |    90.91 |                51 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/Header              |        0 |      100 |        0 |        0 |                   |
  Header.tsx                              |        0 |      100 |        0 |        0 |             10,11 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/List                |        0 |      100 |        0 |        0 |                   |
  List.model.ts                           |        0 |        0 |        0 |        0 |                   |
  List.tsx                                |        0 |      100 |        0 |        0 |          13,14,17 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/ListItem            |        0 |        0 |        0 |        0 |                   |
  ListItem.model.ts                       |        0 |        0 |        0 |        0 |                   |
  ListItem.tsx                            |        0 |        0 |        0 |        0 |             11,12 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SnackBar            |      100 |      100 |      100 |      100 |                   |
  SnackBar.tsx                            |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SocialButton        |      100 |      100 |      100 |      100 |                   |
  SocialButton.model.ts                   |        0 |        0 |        0 |        0 |                   |
  SocialButton.tsx                        |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/pages/Home                           |        0 |      100 |        0 |        0 |                   |
  Home.tsx                                |        0 |      100 |        0 |        0 |             19,20 |
  data.ts                                 |        0 |      100 |      100 |        0 |                 1 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/styles                               |       50 |      100 |      100 |       50 |                   |
  GlobalStyles.tsx                        |        0 |      100 |      100 |        0 |                 4 |
  colors.ts                               |      100 |      100 |      100 |      100 |                   |
  devices.ts                              |        0 |      100 |      100 |        0 |                 1 |
  typography.ts                           |      100 |      100 |      100 |      100 |                   |
------------------------------------------|----------|----------|----------|----------|-------------------|
</pre>
</p>
</details>

**Comment by hayuna** [comment]
<p>Total Coverage: <code>48.48</code></p>
<details><summary>Coverage report</summary>
<p>
<pre>
> frontend@0.1.0 test:coverage /home/runner/work/frontend/frontend
> npm test -- --coverage


> frontend@0.1.0 test /home/runner/work/frontend/frontend
> CI=true react-scripts test --env=jsdom "--coverage"

------------------------------------------|----------|----------|----------|----------|-------------------|
File                                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
------------------------------------------|----------|----------|----------|----------|-------------------|
All files                                 |    48.48 |    85.71 |    36.36 |    48.48 |                   |
 src                                      |        0 |      100 |        0 |        0 |                   |
  App.tsx                                 |        0 |      100 |        0 |        0 |                 6 |
  index.tsx                               |        0 |      100 |      100 |        0 |                 7 |
  react-app-env.d.ts                      |        0 |        0 |        0 |        0 |                   |
 src/assets                               |        0 |        0 |        0 |        0 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/ExampleStorybookComponent |        0 |      100 |        0 |        0 |                   |
  ExampleStorybookComponent.model.ts      |        0 |        0 |        0 |        0 |                   |
  ExampleStorybookComponent.tsx           |        0 |      100 |        0 |        0 |              9,10 |
 src/components/atoms/Button              |    90.91 |    90.91 |      100 |    90.91 |                   |
  Button.model.ts                         |        0 |        0 |        0 |        0 |                   |
  Button.tsx                              |    90.91 |    90.91 |      100 |    90.91 |                51 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/Header              |        0 |      100 |        0 |        0 |                   |
  Header.tsx                              |        0 |      100 |        0 |        0 |             10,11 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/List                |        0 |      100 |        0 |        0 |                   |
  List.model.ts                           |        0 |        0 |        0 |        0 |                   |
  List.tsx                                |        0 |      100 |        0 |        0 |          13,14,17 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/ListItem            |        0 |        0 |        0 |        0 |                   |
  ListItem.model.ts                       |        0 |        0 |        0 |        0 |                   |
  ListItem.tsx                            |        0 |        0 |        0 |        0 |             11,12 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SnackBar            |      100 |      100 |      100 |      100 |                   |
  SnackBar.tsx                            |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SocialButton        |      100 |      100 |      100 |      100 |                   |
  SocialButton.model.ts                   |        0 |        0 |        0 |        0 |                   |
  SocialButton.tsx                        |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/pages/Home                           |        0 |      100 |        0 |        0 |                   |
  Home.tsx                                |        0 |      100 |        0 |        0 |             19,20 |
  data.ts                                 |        0 |      100 |      100 |        0 |                 1 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/styles                               |       50 |      100 |      100 |       50 |                   |
  GlobalStyles.tsx                        |        0 |      100 |      100 |        0 |                 4 |
  colors.ts                               |      100 |      100 |      100 |      100 |                   |
  devices.ts                              |        0 |      100 |      100 |        0 |                 1 |
  typography.ts                           |      100 |      100 |      100 |      100 |                   |
------------------------------------------|----------|----------|----------|----------|-------------------|
</pre>
</p>
</details>

File: src/components/atoms/ListItem/ListItem.styled.ts (hunk 3 above)
Comment (PiotrGrobelak) [reply src/components/atoms/ListItem/ListItem.styled.ts:16]: +

File: src/components/atoms/Header/Header.styled.ts (hunk 4 above)
Comment (PiotrGrobelak) [reply src/components/atoms/Header/Header.styled.ts:8]: +

File: src/pages/Home/Home.tsx (hunk 2 above)
Comment (PiotrGrobelak) [reply src/pages/Home/Home.tsx]: Ok. This solution is working.

**Comment by hayuna** [comment]
<p>Total Coverage: <code>48.48</code></p>
<details><summary>Coverage report</summary>
<p>
<pre>
> frontend@0.1.0 test:coverage /home/runner/work/frontend/frontend
> npm test -- --coverage


> frontend@0.1.0 test /home/runner/work/frontend/frontend
> CI=true react-scripts test --env=jsdom "--coverage"

------------------------------------------|----------|----------|----------|----------|-------------------|
File                                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
------------------------------------------|----------|----------|----------|----------|-------------------|
All files                                 |    48.48 |    85.71 |    36.36 |    48.48 |                   |
 src                                      |        0 |      100 |        0 |        0 |                   |
  App.tsx                                 |        0 |      100 |        0 |        0 |                 6 |
  index.tsx                               |        0 |      100 |      100 |        0 |                 7 |
  react-app-env.d.ts                      |        0 |        0 |        0 |        0 |                   |
 src/assets                               |        0 |        0 |        0 |        0 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/ExampleStorybookComponent |        0 |      100 |        0 |        0 |                   |
  ExampleStorybookComponent.model.ts      |        0 |        0 |        0 |        0 |                   |
  ExampleStorybookComponent.tsx           |        0 |      100 |        0 |        0 |              9,10 |
 src/components/atoms/Button              |    90.91 |    90.91 |      100 |    90.91 |                   |
  Button.model.ts                         |        0 |        0 |        0 |        0 |                   |
  Button.tsx                              |    90.91 |    90.91 |      100 |    90.91 |                51 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/Header              |        0 |      100 |        0 |        0 |                   |
  Header.tsx                              |        0 |      100 |        0 |        0 |             10,11 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/List                |        0 |      100 |        0 |        0 |                   |
  List.model.ts                           |        0 |        0 |        0 |        0 |                   |
  List.tsx                                |        0 |      100 |        0 |        0 |          13,14,17 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/ListItem            |        0 |        0 |        0 |        0 |                   |
  ListItem.model.ts                       |        0 |        0 |        0 |        0 |                   |
  ListItem.tsx                            |        0 |        0 |        0 |        0 |             11,12 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SnackBar            |      100 |      100 |      100 |      100 |                   |
  SnackBar.tsx                            |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SocialButton        |      100 |      100 |      100 |      100 |                   |
  SocialButton.model.ts                   |        0 |        0 |        0 |        0 |                   |
  SocialButton.tsx                        |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/pages/Home                           |        0 |      100 |        0 |        0 |                   |
  Home.tsx                                |        0 |      100 |        0 |        0 |             19,20 |
  data.ts                                 |        0 |      100 |      100 |        0 |                 1 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/styles                               |       50 |      100 |      100 |       50 |                   |
  GlobalStyles.tsx                        |        0 |      100 |      100 |        0 |                 4 |
  colors.ts                               |      100 |      100 |      100 |      100 |                   |
  devices.ts                              |        0 |      100 |      100 |        0 |                 1 |
  typography.ts                           |      100 |      100 |      100 |      100 |                   |
------------------------------------------|----------|----------|----------|----------|-------------------|
</pre>
</p>
</details>

**Comment by PiotrGrobelak** [comment]
I need to set "display: flex" in ItemList to keep the design view on mobile.

**Review by hayuna — APPROVED** [review]

**Comment by hayuna** [comment]
<p>Total Coverage: <code>48.48</code></p>
<details><summary>Coverage report</summary>
<p>
<pre>
> frontend@0.1.0 test:coverage /home/runner/work/frontend/frontend
> npm test -- --coverage


> frontend@0.1.0 test /home/runner/work/frontend/frontend
> CI=true react-scripts test --env=jsdom "--coverage"

------------------------------------------|----------|----------|----------|----------|-------------------|
File                                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
------------------------------------------|----------|----------|----------|----------|-------------------|
All files                                 |    48.48 |    85.71 |    36.36 |    48.48 |                   |
 src                                      |        0 |      100 |        0 |        0 |                   |
  App.tsx                                 |        0 |      100 |        0 |        0 |                 6 |
  index.tsx                               |        0 |      100 |      100 |        0 |                 7 |
  react-app-env.d.ts                      |        0 |        0 |        0 |        0 |                   |
 src/assets                               |        0 |        0 |        0 |        0 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/ExampleStorybookComponent |        0 |      100 |        0 |        0 |                   |
  ExampleStorybookComponent.model.ts      |        0 |        0 |        0 |        0 |                   |
  ExampleStorybookComponent.tsx           |        0 |      100 |        0 |        0 |              9,10 |
 src/components/atoms/Button              |    90.91 |    90.91 |      100 |    90.91 |                   |
  Button.model.ts                         |        0 |        0 |        0 |        0 |                   |
  Button.tsx                              |    90.91 |    90.91 |      100 |    90.91 |                51 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/Header              |        0 |      100 |        0 |        0 |                   |
  Header.tsx                              |        0 |      100 |        0 |        0 |             10,11 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/List                |        0 |      100 |        0 |        0 |                   |
  List.model.ts                           |        0 |        0 |        0 |        0 |                   |
  List.tsx                                |        0 |      100 |        0 |        0 |          13,14,17 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/ListItem            |        0 |        0 |        0 |        0 |                   |
  ListItem.model.ts                       |        0 |        0 |        0 |        0 |                   |
  ListItem.tsx                            |        0 |        0 |        0 |        0 |             11,12 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SnackBar            |      100 |      100 |      100 |      100 |                   |
  SnackBar.tsx                            |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SocialButton        |      100 |      100 |      100 |      100 |                   |
  SocialButton.model.ts                   |        0 |        0 |        0 |        0 |                   |
  SocialButton.tsx                        |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/pages/Home                           |        0 |      100 |        0 |        0 |                   |
  Home.tsx                                |        0 |      100 |        0 |        0 |             19,20 |
  data.ts                                 |        0 |      100 |      100 |        0 |                 1 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/styles                               |       50 |      100 |      100 |       50 |                   |
  GlobalStyles.tsx                        |        0 |      100 |      100 |        0 |                 4 |
  colors.ts                               |      100 |      100 |      100 |      100 |                   |
  devices.ts                              |        0 |      100 |      100 |        0 |                 1 |
  typography.ts                           |      100 |      100 |      100 |      100 |                   |
------------------------------------------|----------|----------|----------|----------|-------------------|
</pre>
</p>
</details>

**Comment by hayuna** [comment]
<p>Total Coverage: <code>48.48</code></p>
<details><summary>Coverage report</summary>
<p>
<pre>
> frontend@0.1.0 test:coverage /home/runner/work/frontend/frontend
> npm test -- --coverage


> frontend@0.1.0 test /home/runner/work/frontend/frontend
> CI=true react-scripts test --env=jsdom "--coverage"

------------------------------------------|----------|----------|----------|----------|-------------------|
File                                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
------------------------------------------|----------|----------|----------|----------|-------------------|
All files                                 |    48.48 |    85.71 |    36.36 |    48.48 |                   |
 src                                      |        0 |      100 |        0 |        0 |                   |
  App.tsx                                 |        0 |      100 |        0 |        0 |                 6 |
  index.tsx                               |        0 |      100 |      100 |        0 |                 7 |
  react-app-env.d.ts                      |        0 |        0 |        0 |        0 |                   |
 src/assets                               |        0 |        0 |        0 |        0 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/ExampleStorybookComponent |        0 |      100 |        0 |        0 |                   |
  ExampleStorybookComponent.model.ts      |        0 |        0 |        0 |        0 |                   |
  ExampleStorybookComponent.tsx           |        0 |      100 |        0 |        0 |              9,10 |
 src/components/atoms/Button              |    90.91 |    90.91 |      100 |    90.91 |                   |
  Button.model.ts                         |        0 |        0 |        0 |        0 |                   |
  Button.tsx                              |    90.91 |    90.91 |      100 |    90.91 |                51 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/Header              |        0 |      100 |        0 |        0 |                   |
  Header.tsx                              |        0 |      100 |        0 |        0 |             10,11 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/List                |        0 |      100 |        0 |        0 |                   |
  List.model.ts                           |        0 |        0 |        0 |        0 |                   |
  List.tsx                                |        0 |      100 |        0 |        0 |          13,14,17 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/ListItem            |        0 |        0 |        0 |        0 |                   |
  ListItem.model.ts                       |        0 |        0 |        0 |        0 |                   |
  ListItem.tsx                            |        0 |        0 |        0 |        0 |             11,12 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SnackBar            |      100 |      100 |      100 |      100 |                   |
  SnackBar.tsx                            |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SocialButton        |      100 |      100 |      100 |      100 |                   |
  SocialButton.model.ts                   |        0 |        0 |        0 |        0 |                   |
  SocialButton.tsx                        |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/pages/Home                           |        0 |      100 |        0 |        0 |                   |
  Home.tsx                                |        0 |      100 |        0 |        0 |             19,20 |
  data.ts                                 |        0 |      100 |      100 |        0 |                 1 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/styles                               |       50 |      100 |      100 |       50 |                   |
  GlobalStyles.tsx                        |        0 |      100 |      100 |        0 |                 4 |
  colors.ts                               |      100 |      100 |      100 |      100 |                   |
  devices.ts                              |        0 |      100 |      100 |        0 |                 1 |
  typography.ts                           |      100 |      100 |      100 |      100 |                   |
------------------------------------------|----------|----------|----------|----------|-------------------|
</pre>
</p>
</details>

**Pull request merged**